.github/workflows/on_pr.yaml

```
@@ -0,0 +1,46 @@
+name: On Pull Request
+
+on: pull_request
```

Vocês podem testar se assim funciona?
              
```suggestion
on:
  push: 
    branches: [ main ]
  pull_request:
    branches: [ main ]
```
Assim a gente consegue também automatizar os testes quando o código for para a Main!